### PR TITLE
Features/user profile

### PR DESCRIPTION
**Pull request by AndreyAnokhin**
Change base template to support boostrap4.
Also, you need to install:
```
pip install Flask-Bootstrap4
```

### CODE REVIEW

File: app/templates/profile.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <div class="text-center">
+        {% with messages = get_flashed_messages() %}
```
Comment (CHuKeR) [inline app/templates/profile.html]: We don't need flash here, there are in base template

File: requirements.txt [hunk 2]
```diff
@@ -1,10 +1,9 @@
 alembic==1.0.11
 Click==7.0
 dominate==2.4.0
-entrypoints==0.3
-flake8==3.7.8
 Flask==1.1.1
 Flask-Bootstrap==3.3.7.1
```
Comment (CHuKeR) [inline requirements.txt:5]: maybe delete bootstap3?

**Review by CHuKeR — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 2 above)
Comment (AndreyAnokhin) [reply requirements.txt:5]: Flask-Bootstrap4 just adds bootstrap4 styles to Flask-Bootstrap module. The app won't be work without main Flask-Bootstrap module.

File: app/templates/profile.html (hunk 1 above)
Comment (AndreyAnokhin) [reply app/templates/profile.html]: ok, you're right

**Pull request merged**